modules/web/js/ballerina/components/package-definition.jsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+
+import {packageDefinition} from '../configs/designer-defaults';
+import './package-definition.css';
+import {getCanvasOverlay} from '../configs/app-context';
+import {renderTextBox} from './text-input';
+
+class PackageDefinition extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.handlePackageNameInput = this.handlePackageNameInput.bind(this);
+    }
+
+    handlePackageNameInput(input) {
+        //debugger;
+        this.props.model.setPackageName(input);
+    }
+
+    handlePackageNameClick(e) {
+        const model = this.props.model;
+        const bBox = model.viewState.bBox;
+        const headerPadding = packageDefinition.header.padding;
+
+        const textBoxBBox = {
+            x: bBox.x + headerPadding.left + 75,
+            y: bBox.y + 15
+        }
+
+        renderTextBox(textBoxBBox, this.handlePackageNameInput, model.getPackageName());
+    }
+
+    render() {
+        const model = this.props.model;
+        const bBox = model.viewState.bBox;
+        const packageName = model._packageName || "";
+        const headerHeight = packageDefinition.header.height;
+        const headerPadding = packageDefinition.header.padding;
+
+        return (
+            <g>
+                <rect x={ bBox.x } y={ bBox.y } width={ bBox.w } height={ headerHeight } rx="0" ry="0" className="package-definition-header"/>
+                <text x={ bBox.x + headerPadding.left } y={ bBox.y + headerHeight/2 } className="package-definition-text">Package</text>
```

Comment:
Is there a formatting issue here 'headerHeight/2'  ? I think there should be like 'headerHeight / 2'